presto-orc/src/main/java/com/facebook/presto/orc/reader/SelectiveStreamReader.java

```diff
@@ -58,7 +59,7 @@ int read(int offset, int[] positions, int positionCount)
      * May be called multiple times after each read(). On repeated invocation, the list of positions
      * must be a strict subset of positions passed in the previous invocation.
      */
-    Block getBlockView(int[] positions, int positionCount);
+    BlockLease getBlockView(int[] positions, int positionCount);
```

Comment:
Maybe update the javadoc by refining the wording from https://github.com/prestodb/presto/pull/13066#discussion_r305907884 to better explain the interface.